int32 class_id
string class_name
float32 score
float32 y_min
float32 x_min
float32 y_max
float32 x_max
